sorts/stooge_sort.py

```diff
@@ -1,11 +1,17 @@
 def stooge_sort(arr):
     """
-    >>> arr = [2, 4, 5, 3, 1]
-    >>> stooge_sort(arr)
-    >>> print(arr)
-    [1, 2, 3, 4, 5]
+    Examples:
+    >>> stooge_sort([0, 5, 3, 2, 2])
+    [0, 2, 2, 3, 5]
+
+    >>> stooge_sort([])
+    []
+
+    >>> stooge_sort([-2, -5, -45])

```

Comment:
Good but let’s mix positive int, positive float, zero, negative int, and negative float all in one test.